src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -21,6 +21,12 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
   const { title, authorName, categories } = posterVenue.poster ?? {};
 
   const venueId = posterVenue.id;
+  const posterId = venueId.replace("poster", "");
+  // TODO: the posterId doesn't correspond to abs ID. We will need proper URL in DB.
+  // This version is just to demonstrate possibility
+  const posterAbsUrl =
+    "https://ww4.aievolution.com/hbm2101/index.cfm?do=abs.viewAbs&src=ext&abs=" +
+    posterId;
```

Comment:
Doesn't it make more sense to just store it in `poster`? Looks like an unnecessary complex implementation

Reply:
See TODO comment above the const definition :-)

Reply:
@yarikoptic oops, my bad. Haha, it means that we are on the same page though 😄